lms/static/js/spec/staff_debug_actions_spec.js

```diff
@@ -31,6 +31,15 @@ define([
                 });
             });
 
+            describe('getURL ', function() {
+                it('defines url to courseware ajax entry point on instructor page', function() {
+                    spyOn(StaffDebug, 'getCurrentUrl')
+                      .and.returnValue('/courses/edx_demo_course/instructor#view-open_response_assessment');
+                    expect(StaffDebug.getURL('rescore_problem'))
+                      .toBe('/courses/edx_demo_course/instructor/api/rescore_problem');
+                });
+            });
+
```

Comment:
How is it testing the functionality that we added?

Reply:
@AhtishamShahid Was only the `rescore_problem` URL causing problem? Because if not, add some other cases here too.